packages/util/src/units.ts

```diff
@@ -1,6 +1,9 @@
 import { BIGINT_0, BIGINT_1 } from './constants.js'
+
 /** Easy conversion from Gwei to wei */
 export const GWEI_TO_WEI = BigInt(1000000000)
+export const WEI_TO_EITHER = BigInt(10 ** 18)
```

Comment:
This is inconsistent, the above constant which was already there is the conversion rate from Gwei to Wei (the multiplier. So this should be `ETHER_TO_WEI` (there also is a typo here `EITHER` -> `ETHER`).